# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.h
# Commit: 8b2be3d7ece9
# Full Hash: 8b2be3d7ece96e3ea35ea4f7b4a9cf7d3559012e
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2018-11-27 21:53:40
# Regressor Bug: 1493400
# File Overlap Count: 1
# Description:
#   Bug 1493400 - Implement platform decoder for dav1d. r=jya
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D12164
# ==============================================================================

diff -r 5c4bf474ddb3 -r 8b2be3d7ece9 modules/libpref/init/StaticPrefList.h
--- a/modules/libpref/init/StaticPrefList.h	Sat Nov 24 01:53:31 2018 +0000
+++ b/modules/libpref/init/StaticPrefList.h	Tue Nov 27 14:18:58 2018 +0000
@@ -1397,6 +1397,11 @@
    MediaAv1Enabled,
   RelaxedAtomicBool, false
 )
+VARCACHE_PREF(
+  "media.av1.use-dav1d",
+   MediaAv1UseDav1d,
+  RelaxedAtomicBool, false
+)
 
 VARCACHE_PREF(
   "media.flac.enabled",